src/main/java/com/google/codeu/servlets/AboutMeServlet.java

```diff
@@ -0,0 +1,74 @@
+package com.google.codeu.servlets;
+
+import java.io.IOException;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.codeu.data.Datastore;
+import com.google.codeu.data.User;
+
+import org.jsoup.Jsoup;
+import org.jsoup.safety.Whitelist;
+
+/**
+* Third Test - Fetches and saves user data
+*/
+@WebServlet("/about")
+public class AboutMeServlet extends HttpServlet{
+  private Datastore datastore;
+
+  @Override
+  public void init(){
+    datastore = new Datastore();
+  }
+
+  /**
+  * Responds with "about me" section for the user
+  */
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response)
+    throws IOException{
+
+      response.setContentType("text/html");
+
+      String user = request.getParameter("user");
+      if(user == null || user.equals("")){
```

Comment:
minor style nit here and elsewhere: Add space between 'if' and '(' and between the ')'  and '{'. Usually this is mainly to fit in with the surrounding code so your changes don't stand out.

Reply:
Reformatted in Commits 6c5c5b8c56dd2a03548a89c3b9e743e249584730 and 2b653b685d4a583914bca81d3cbdaf3ca5db695a